README.md

```diff
@@ -1 +1,152 @@
-# musical-concert-capstone
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#Musical-Concert-Capstone)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#html-css)
+    - [Key Features](#home-page)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#amen-musingarimi)
+- [🔭 Future Features](#the other pages)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#microverse)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Microverse-Portfolio-Project] <a name="Musical-Concert-Capstone"></a>
+
+**[Musical-Concert-Capstone]** is a website that basically advertises a music concert that will take place in future. It provide the required information about the concert, that is brief description, the program plan, the artists to perfom and also the partnering companies for the event.
+
+## 🛠 Built With <a name="html, css and JavaScript"></a>
+
+### Tech Stack <a name="html, css and JavaScript"></a>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Social_Media_links]**
+- **[navigation_bar]**
+- **[home_page]**
+- **[about_page]**
+- **[main_program_section]**
+- **[featured_artists_section]**
+- **[partners_section]**
+- **[footer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://amen-musingarimi.github.io/musical-concert-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need any code editor, for example Visual Studio.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Run tests
+
+To run tests, run the following command: npx hint . and npx stylelint "\*_/_.{css,scss}" and npx eslint .
+
+### Deployment
+
+You can deploy this project using Github pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
```

Comment:
- [ ] Kindly fill these sections with the required commands to run your project on a local machine :+1: If any sub section is not necessary then kindly remove that section :blush: